graphql/schema.graphql

```diff
@@ -487,6 +486,7 @@ type Patch {
   status: String!
   variants: [String!]!
   tasks: [String!]!
+  childPatches: [ChildPatches]
```

Comment:
we can make [ChildPatches] non-nullable since we don't return nil unless an error is returned in the resolver

Reply:
Most patches will not have [ChildPatches], what is the benefit of making it non-nullable?